ruff_lsp/server.py

```diff
@@ -364,25 +367,33 @@ def _get_severity(code: str) -> DiagnosticSeverity:
 CODE_REGEX = re.compile(r"[A-Z]+[0-9]+")
 
 
-@LSP_SERVER.feature(TEXT_DOCUMENT_HOVER)
-async def hover(params: HoverParams) -> Hover | None:
-    """LSP handler for textDocument/hover request."""
-    document = LSP_SERVER.workspace.get_text_document(params.text_document.uri)
-    match = NOQA_REGEX.search(document.lines[params.position.line])
+def _parse_codes(line: str) -> list[tuple(str, int, int)]:
```

Comment:
Could we have a `NamedTuple` / `dataclass` / `TypedDict` instead of `(str, int, int)`? It's hard to know what those represents otherwise.